### PR TITLE
update/SOF 7461

### PR DESCRIPTION
**Pull request by VsevolodX**
- upsdate: add get_materials wrapper
- update: add set_materials wrapper

### CODE REVIEW

File: other/materials_designer/create_interface_with_min_strain_zsl.ipynb [hunk 1]
```diff
@@ -50,8 +50,27 @@
    "source": [
```
Comment (timurbazhirov) [inline other/materials_designer/create_interface_with_min_strain_zsl.ipynb:50]: <pre class="ql-syntax" spellcheck="false">Line #4.    FILM_INDEX = 1
</pre><p> </p><p><code>FILM_INDEX = 1  # Index in the list of materials, to access as materials[FILM_INDEX]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (timurbazhirov) [inline other/materials_designer/create_interface_with_min_strain_zsl.ipynb:50]: <pre class="ql-syntax" spellcheck="false">Line #8.        print("Please select a film material. Film is set to substrate.")
</pre><p>Film material not found. Re-using substrate material as film.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: other/materials_designer/create_interface_with_no_strain_matching.ipynb [hunk 2]
```diff
@@ -43,7 +43,26 @@
     "# Enable interactive selection of terminations via UI prompt\n",
```
Comment (timurbazhirov) [inline other/materials_designer/create_interface_with_no_strain_matching.ipynb:43]: <pre class="ql-syntax" spellcheck="false">Line #21.    TERMINATION_PAIR_INDEX = 0
</pre><p>TERMINATION_PAIR_INDEX # Will be overriden in interactive selection is used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: other/materials_designer/create_island_defect.ipynb [hunk 3]
```diff
@@ -0,0 +1,1079 @@
+{
```
Comment (timurbazhirov) [inline other/materials_designer/create_island_defect.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.    materials = list(map(Material, globals()["materials_in"]))
</pre><p>Let's clean up the outputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: other/materials_designer/create_interface_with_min_strain_zsl.ipynb (hunk 1 above)
Comment (VsevolodX) [reply other/materials_designer/create_interface_with_min_strain_zsl.ipynb:50]: <p>ok</p>

Comment (VsevolodX) [reply other/materials_designer/create_interface_with_min_strain_zsl.ipynb:50]: <p>ok</p>

File: other/materials_designer/create_interface_with_no_strain_matching.ipynb (hunk 2 above)
Comment (VsevolodX) [reply other/materials_designer/create_interface_with_no_strain_matching.ipynb:43]: <p>ok</p>

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**